src/components/ui/Checkbox/Checkbox.tsx

```diff
@@ -1,20 +1,22 @@
 import { Check } from 'lucide-react'
-import React from 'react'
-import { Checkbox as ReactAriaCheckbox, type CheckboxProps} from 'react-aria-components'
+import React, { ReactNode } from 'react'
+import { Checkbox as ReactAriaCheckbox, type CheckboxProps as AriaCheckboxProps } from 'react-aria-components'
 
 const CLASS_NAME = 'Layer__Checkbox'
 
-export function Checkbox({children, ...props}: Omit<CheckboxProps, 'className'>) {
+export type CheckboxProps = Omit<AriaCheckboxProps, 'className' | 'children'> & { children?: ReactNode | ReactNode[] }
```

Comment:
Not sure if it's because of the latest `react-aria-components` release, but for some reasons `AriaCheckboxProps`'s `children` type couldn't match here